core/src/consensus/stake/action_data.rs

```diff
@@ -410,28 +410,28 @@ impl IntoIterator for Validators {
 
 #[derive(Default, Debug, PartialEq)]
 pub struct IntermediateRewards {
-    previous: BTreeMap<Address, u64>,
-    current: BTreeMap<Address, u64>,
+    pub(super) current: BTreeMap<Address, u64>,
+    pub(super) calculated: BTreeMap<Address, u64>,
```

Comment:
Could you explain to me why you used `pub(super)`?

Reply:
It came from the original PR in CodeChain since `IntermediateRewards` was wrapped inside a module.
We don't need it anymore. I'll remove this.